Python/ceval.c

```diff
@@ -818,6 +819,10 @@ _PyEval_EvalFrameDefault(PyThreadState *tstate, _PyInterpreterFrame *frame, int
             DISPATCH();
         }
         if (_PyOpcode_Caches[original_opcode]) {
+            assert(original_opcode != POP_JUMP_IF_FALSE);
+            assert(original_opcode != POP_JUMP_IF_TRUE);
+            assert(original_opcode != POP_JUMP_IF_NONE);
+            assert(original_opcode != POP_JUMP_IF_NOT_NONE);
```

Comment:
@markshannon Same thing here.

Reply:
(I think -- it doesn't actually fail without this, AFAICT, but it seems to make sense.)

Reply:
The `|= 1` will work here too.